### PR TITLE
Adding tokenHelper which replaces jsonwebtoken package and reduces bundle size

### PR DESCRIPTION
**Pull request by phongcao**
This helps reduce the bundle (`./reflect-bundle.js` file) size from 2.16 MiB to 1.51 MiB.

### CODE REVIEW

**Review by polatengin — APPROVED** [review]
🥳💯

File: RetrospectiveExtension.Frontend/utilities/tokenHelper.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ */
+export const decodeJwt = (token: string): null | JwtPayload => {
+  try {
+    const base64Main = token.split('.')[1];
+    const decodedMain = Buffer.from(base64Main, 'base64').toString();
+
+    return JSON.parse(decodedMain);
+  }
+  catch (e) {
+    return null;
```
Comment (argoldbe) [inline RetrospectiveExtension.Frontend/utilities/tokenHelper.tsx:26]: Should we track this exception via app insights?

Comment (phongcao) [reply RetrospectiveExtension.Frontend/utilities/tokenHelper.tsx:26]: Yes, but I think that we should address it in a different story, something like `Tracking all errors with App Insights`. Right now if you search for `throw new Error` in the code, there are many places that we're doing that and they're good candidates for tracking exceptions with AI.

What do you think? @argoldbe @NarmathaBala

Comment (argoldbe) [reply RetrospectiveExtension.Frontend/utilities/tokenHelper.tsx:26]: Sounds good - maybe just add a TODO for now.

**Review by argoldbe — APPROVED** [review]

File: RetrospectiveExtension.Frontend/utilities/tokenHelper.tsx (hunk 1 above)
Comment (phongcao) [reply RetrospectiveExtension.Frontend/utilities/tokenHelper.tsx:26]: Added.

**Pull request merged**